outreach/README.md

```diff
@@ -14,11 +14,47 @@ Read is used to list all records of a given type. For example, if you want to li
 // Call Read to list records in a users object
 res, err := client.Read(context.TODO(),common.ReadParams{
 		ObjectName: "users",
-        })
+})
 if err != nil {
 	log.Fatal(err)
 }
 
 ```
 
+## Write
+Write is used to create/update objects in outreach connector. For an instance creating an emailAddress object, you use the Write methid and `emailAddress` object   
+
+### Example usage
+```
+
+type Attribute struct {
+	Email     string `json:"email"`
+	EmailType string `json:"emailType"`
+	Order     int    `json:"order"`
+	Status    string `json:"status"`
+}
+
+type EmailAddress struct {
+	Attributes Attribute `json:"attributes"`
+	Type       string    `json:"type"`
+}
+
+// Create an outreach connector instance (any method you prefer)
+// This Assumes you named the instance client
+
+config := common.WriteParams{
+		ObjectName: "emailAddresses",
+		RecordData: EmailAddressUpdate{
+			Attributes: patt,
```

Comment:
where's `patt` initialized in this code sample?

Reply:
Thanks, was using the update sample and didn't include tha attributes. Reverting to creating sample.